install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Syntax: install.sh <Root Dir> <Python SiteLib> <Prefix>
+set -x 
+
+export rootdir=$1
+export sitelib=$2
+export prefix=$3
+#python setup.py install --skip-build --prefix=$prefix
+python setup.py install --skip-build --root=$rootdir --prefix=$prefix
+rm -rf $rootdir/$sitelib/redfish/old
```

Comment:
Fedora package rules request to remove the egg info. I think this is to ensure that you will recreate them with setup and have the latest one.